tests/test_classifier/test_prcurve.py

```diff
@@ -262,3 +264,56 @@ def test_no_scoring_function(self):
         oz = PrecisionRecallCurve(FakeClassifier())
         with pytest.raises(ModelError, match="requires .* predict_proba or decision_function"):
             oz._get_y_scores(self.binary.X.train)
+
+    def test_custom_iso_f1_scores(self):
+
+        iris = load_iris()
+        x = iris.data[:, :] 
+        y = iris.target
+
+        vals = (0.1,0.6,0.3,0.9,0.9)
```

Comment:
This is a very, very good test! I like the out of order numbers and duplicate 0.9!